packages/shared/src/components/modals/CommentBox.tsx

```diff
@@ -36,12 +36,14 @@ export interface CommentBoxProps {
   sendComment: (event: MouseEvent | KeyboardEvent) => Promise<void>;
   onInput?: (value: string) => unknown;
   onKeyDown: (
-    e: KeyboardEvent<HTMLTextAreaElement>,
-    defaultCallback?: KeyboardEventHandler<HTMLTextAreaElement>,
+    e: KeyboardEvent<HTMLDivElement>,
+    defaultCallback?: KeyboardEventHandler<HTMLDivElement>,
   ) => unknown;
   post: Post;
 }
 
+const cleanupText = (text: string) => text.replaceAll?.('\xa0', ' ') || text;
```

Comment:
It might be unclear that `xa0` is empty space. perhaps we should rename cleanupText to state cleanupEmptySpaces?

But this is very optional, happy either way.

Reply:
Second note, perhaps we can put this in general utils? as we most likely will need it in other places later on.

Reply:
Totally agree with your suggested name. Will work on it real quick! ⛴️